index.js

```diff
@@ -55,14 +55,33 @@ module.exports = postcss.plugin('postcss-increase-specificity', function(options
 	var opts = objectAssign({}, defaults, options);
 
 	return function(css) {
-		css.walkRules(function(rule) {
-			// Avoid adding additional selectors (stackableRoot) to descendant rules of @keyframe {}
-			// i.e. `from`, `to`, or `{number}%`
-			var isInsideKeyframes = rule.parent.type === 'atrule' && rule.parent.name === 'keyframes';
+		let isExcluded = false;
+		css.walk(function(node) {
 
-			if(!isInsideKeyframes) {
-				increaseSpecifityOfRule(rule, opts);
+			if (node.type === 'comment') {
+
+				const commentStart = 'no important-start';
+				const commentEnd = 'no important-end';
+
+				if(node.text === commentStart) {
```

Comment:
Do we have to worry about comparing whitespace here? Perhaps we need to `node.text.trim()`?